signer/src/config/mod.rs

```diff
@@ -24,6 +24,10 @@ use crate::stacks::wallet::SignerWallet;
 mod error;
 mod serialization;
 
+/// The minimum number of blocks between the hight at which the reclaim script path can
+/// be spent and the current chain tip for the deposit to be considered acceptable.
+pub const MINIMUM_RECLAIM_PROXIMITY_TO_CHAIN_TIP: u16 = 10;
```

Comment:
With #744 merged, do we still need this or can we reuse https://github.com/stacks-network/sbtc/blob/main/signer/src/lib.rs#L58 ?

Reply:
We can reuse that

Reply:
Done.